src/core/curlify.js

```diff
@@ -25,15 +25,14 @@ export default function curl( request ){
   if ( headers && headers.size ) {
     for( let p of request.get("headers").entries() ){
       let [ h,v ] = p
-      type = v
+      headerValues[v] = h
       curlified.push( "-H " )
       curlified.push( `"${h}: ${v}"` )
     }
   }
 
   if ( request.get("body") ){
-
-    if(type === "multipart/form-data" && ["POST", "PUT", "PATCH"].includes(request.get("method"))) {
+    if (headerValues["multipart/form-data"] && ["POST", "PUT", "PATCH"].includes(request.get("method"))) {
```

Comment:
The code itself has an obvious bug, demonstrated by following non-failing test case:
```js
context('given request with Authorization header of multipart/form-data value', function () {
  specify('should not consider request multipart', function () {
    let file = new win.File()
    file.name = "file.txt"
    file.type = "text/plain"

    let req = {
      url: "http://example.com",
      method: "POST",
      headers: {
        "Authorization": "multipart/form-data",
        "x-custom-name": "any-value"
      },
      body: {
        id: "123",
        file
      }
    }
  
   let curlified = curl(Im.fromJS(req))
    ...what are we expecting here...?
});